Makefile

```diff
@@ -300,6 +300,11 @@ lint-js: $(NODE_MODULES)
 lint-css: $(NODE_MODULES)
 	$(NODE_MODULES)/.bin/sass-lint -q -v
 
+.PHONY: lint-components
+lint-components:
+	 @./scripts/inspect-components validate --path jujugui/static/gui/src/ --short || \
+	 	echo "^^^ Ping frankban if those are not actual errors.\n"
```

Comment:
Nice. :smile:

Do we want to specify a channel for non canonical contributors? Or at this point are we just admitting that we're the only one's working on this? :stuck_out_tongue_closed_eyes: